float32[] t
float32[] pres_tank
float32[] pres_act1_kr
float32[] pres_act2_hr
float32[] pres_act3_hl
float32[] pres_act4_kl
